data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+<p>And his exordium, as I have already said, is most admirable; embracing the creation of the world, under the idea that the law corresponds to the world and the world to the law, and that a man who is obedient to the law, being, by so doing, a citizen of the world, arranges his actions with reference to the intention of nature, in harmony with which the whole universal world is regulated.</p>
+</div>
+
+<div type="textpart" subtype="section" n="4">
+<p>Accordingly no one, whether poet or historian, could ever give expression in an adequate manner to the beauty of his ideas respecting the creation of the world; for they surpass all the power of language, <choice><sic>and amaze.</sic><corr>and amaze</corr></choice> our hearing, being too great and venerable to be adapted to the senses of any created being.</p>
+</div>
+
+<div type="textpart" subtype="section" n="5">
+<p>That, however, is not a reason for our yielding to indolence on the subject, but rather from our affection for the Deity we ought to endeavour to exert ourselves even beyond our powers in describing them: not as having much, or indeed anything to say of
+<pb n="2"/>
+our own, but instead of much, just a little, such as it may be probable that human intellect may attain to, when wholly occupied with a love of and desire for wisdom.
+</p>
+</div>
+
+<div type="textpart" subtype="section" n="6">
+<p>For as the smallest seal receives imitations of <choice><sic>things or</sic><corr>things of</corr></choice> colossal magnitude when engraved upon it, so perchance in some instances the exceeding beauty of the description of the creation of the world as recorded in the Law, overshadowing with its brilliancy the souls of those who happen to meet with it, will be delivered to a more concise record after these facts have been first premised which it would be improper to pass over in silence.
+<milestone unit="chapter" n="2"/></p>
+</div>
+
+<div type="textpart" subtype="section" n="7">
+<p>For some men, admiring the world itself rather than the Creator of the world, have represented it as existing without any maker, and eternal; and as impiously as falsely have represented God as existing in a state of complete inactivity, while it would have been right on the other hand to marvel at the might of God as the creator and father of all, and to admire the world in a degree not exceeding the bounds of moderation.
+</p>
+</div>
+
+<div type="textpart" subtype="section" n="8">
+<p>But Moses, who had early reached the very summits of philosophy, <ref n="2.1"/> and who had learnt from the oracles of God the most numerous and important of the principles of nature, was well aware that it is indispensable that in all existing things there must be an active cause, and a passive subject; and that the active cause is the intellect of the universe, thoroughly unadulterated and thoroughly unmixed, superior to virtue and superior to science, superior even to abstract good or abstract beauty;</p>
```

Comment:
Don't know what the `<ref>` is here. Perhaps the note? Most notes are inline, and I think if we want them this way, it would be something like `xml:id` not simply `id`